README.md

```diff
@@ -0,0 +1,115 @@
+[![Build Status](https://app.bitrise.io/app/e92e109959b2fd4a/status.svg?token=RK3sadyzv8puC80GhNLq3w&branch=develop)](https://app.bitrise.io/app/e92e109959b2fd4a)
+
+C0T0 - simply networking library for iOS based on NSURLSession 
+## Installation
+
+### CocoaPods
+
+To integrate C0T0 into your Xcode project using CocoaPods, specify it in your `Podfile`:
+
+```ruby
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '10.0'
+use_frameworks!
+
+target '<target name>' do
+  pod 'C0T0', '~> 0.1.0'
+end
+
+```
+
+Then, run the following command:
+
+```bash
+$ pod install
+```
+
+## Usage
+
+### Making a simple GET request
+
+Create an object of `SessionConfiguration` with specified host.
+
+If you're interested in logging requests, responses and errors, just enable logging in `SessionConfiguration` `init` method. By default logging is disable (also the default value is set so you can skip it in init method).
+
+```swift 
+    private let sessionConfiguration = SessionConfiguration(host: "https://api.github.com",
+                                                            loggingEnabled: true)
+```
+
+Next create instance of `ApiService`
+```swift
+    private lazy var apiService = ApiService(configuration: sessionConfiguration)
+```
+
+Remember to retain the `apiService` object instead of creating local variable inside a function's scope, otherways your object may be released before the response will be received.
+
+Here's an example of `requestRepositories(for ogranization)` function with setting repositories array for success or showing the error alert in case of failure. For parsing the results we're using `Codable` models. So for the Result you have to provide a generic type which is conforming to `Codable` protocol.
```

Comment:
I think method name is `requestRepositories(for:)`, `organization` is just a parameter name and it doesn't count to method name